slack_gpt_bot.py

```diff
@@ -192,7 +192,7 @@ def handle_app_mentions(body, context):
             thread_ts=thread_ts,
             user=user.username, 
             email=user.email,
-            request=messages[1:],   #field 0 is something that gets added as part of process_conversation_history that we don't need
+            request=messages[-1],   #field 0 is something that gets added as part of process_conversation_history that we don't need
```

Comment:
The last element in the message array will never be empty right?  I don't think it would be since its initiated by the first message when a thread is created.  Just want to double check so we don't throw an out of bounds exception.

Reply:
I don't think so. The 0th element is always that system message that we've been ignoring ("You are an AI Assistant...") The function in utils.py, `process_conversation_history`, handles the creation of the list. If the message text from the request isn't valid, then nothing gets appended to the list.  So if the request was really jenky and the code could't process it, we'd end displaying the system message in the log. I can add a task to look into logging around the message processing so that we error if there is an issue there.

Reply:
Thanks for the clarification.  Lets leave it as is for now then.  Should be just fine.